docs/roadmap/rm-2020.md

```diff
@@ -0,0 +1,79 @@
+# 2020 Eventing Roadmap
+
+## References
+
+[Personas and User Stories](../personas.md):
+
+Knative Eventing caters to the following personas when developing different
+features
+
+1. Event consumer (developer)
+1. Event producer
+1. System Integrator
+1. Contributors
+
+## Areas of Interest and Requirements
+
+1. **API Maturity**. Description TBD
+1. **Production-ready Brokers**. Description TBD
+1. **Knative Inbound Authorization**. Description TBD
+1. **Serverless Ecosystem**. Description TBD
+1. **Test and Release**. Description TBD
+1. **Multi-Tenancy**. Description TBD
+
+### API Maturity
+
+1. **Define Broker spec and conformance tests** including data plane
+   specification https://github.com/knative/eventing/issues/2614. (issue TBD)
+1. **Graduate to V1 APIs** This also mplies increasing of conformance test for
+   channels, broker and sources. (issue TBD)
+
+### Production-ready Brokers
+
+1. **Minimal Footprint (CPU/RAM)** The underlying platform components have a
+   minimal (near zero) footprint (CPU/RAM) in the cluster when eventing is not
+   being used. (issue TBD)
+1. **Observability** Improvements to E2E tracing, metrics, and logs. (issue TBD)
+1. **Lossless upgrades** Brokers never go unavailable during upgrade and no
+   events get lost. (issue TBD)
+
+### Knative Inbound Authorization
+
+1. **Application Level Authorization** (Not limited to eventing) Standardize a
+   Knative way to do application level authorization and make the policy
+   implementation/provider extensible (e.g. GCP IAM, Azure AD, etc.). (issue
+   TBD)
+1. **Unauthorized Requests Denial** (Not limited to eventing) For Knative
```

Comment:
For HTTP, CloudEvents defines this: https://github.com/cloudevents/spec/blob/v1.0/http-webhook.md#4-abuse-protection